Custom build is failing

The output
```
prompt> npm run-script dist

> zepto@1.2.0 dist path-here\zepto.js
> coffee make dist

dist/zepto.js: 48.5 KiB
path-here\zepto.js\make:171
    compressor = uglify.Compressor();
                        ^

TypeError: uglify.Compressor is not a function
    at minify (path-here\zepto.js\make:171:25)
    at Function.target.minify (path-here\zepto.js\make:92:14)
    at Object.target.(anonymous function) [as minify] (path-here\zepto.js\node_modules\shelljs\make.js:28:26)
    at Function.target.dist (path-here\zepto.js\make:56:12)
    at Object.target.(anonymous function) [as dist] (path-here\zepto.js\node_modules\shelljs\make.js:28:26)
    at path-here\zepto.js\node_modules\shelljs\make.js:38:20
    at Array.forEach (<anonymous>:null:null)
    at Timeout._onTimeout (path-here\zepto.js\node_modules\shelljs\make.js:36:10)
    at listOnTimeout (timers.js:324:15)
    at processTimers (timers.js:268:5)

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! zepto@1.2.0 dist: `coffee make dist`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the zepto@1.2.0 dist script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```
Win10, node 11.2

Reverting to the version of uglify from the original repo fixes the issue.

--- a/package.json
+++ b/package.json
@@ -12,9 +12,9 @@
     "license": "MIT",
     "devDependencies": {
         "coffeescript": "^2.4.1",
-        "express": ">=3.11.0",
+        "express": "^4.17.1",
         "shelljs": "0.1.x",
-        "uglify-js": ">=2.6.0"
+        "uglify-js": "^2.8.29"
     },
     "dependencies": {
         "body-parser": "^1.19.0"

